tests/system/env.yml

```diff
@@ -0,0 +1,14 @@
+# The mlrun-api URL. e.g. https://mlrun-api.default-tenant.app.hedingber-28-1.iguazio-cd2.com
```

Comment:
Why is this a yml file ? env key=value shouldn't be declared verbatim in a yaml (https://docs.docker.com/compose/env-file/)